packages/typespec-vscode/src/emit/emit-quick-pick-item.ts

```diff
@@ -0,0 +1,15 @@
+import vscode from "vscode";
+import { EmitterKind } from "./emitter.js";
+
+export interface EmitQuickPickItem extends vscode.QuickPickItem {
+  language: string;
+  package: string;
+  version?: string;
+  fromConfig: boolean;
+  outputDir?: string;
+  emitterKind: EmitterKind;
+}
+
+export interface TypeSpecProjectPickItem extends vscode.QuickPickItem {
```

Comment:
I guess the code should still work if you remove this definition, it's not worthwhile when the impact of the type is just a small part of function.